tests/test_context.py

```diff
@@ -213,17 +213,63 @@ def setUpClass(cls):
         component.registry.register(name="parent_with_args", component=ParentComponentWithArgs)
         component.registry.register(name="variable_display", component=VariableDisplay)
 
-    def test_parent_args_can_be_drawn_from_context(self):
-        template = Template(
-            "{% load component_tags %}{% component_dependencies %}"
-            "{% component 'parent_with_args' parent_value=parent_value %}"
-            "{% endcomponent %}"
+    @override_settings(
+        COMPONENTS={
+            "context_behavior": "django",
+        }
+    )
+    def test_parent_args_can_be_drawn_from_context__django(self):
```

Comment:
For tests that were failing after changing the default `context_behavior`, I created tests for both variants, suffixed test names with `__django` or `__isolated`.